data/campaigns/Winds_of_Fate/scenarios/05x_Unexpected_Arrival.cfg

```diff
@@ -0,0 +1,329 @@
+            mirror=yes
+            message= _ "We have not yet the provisions for three flights within this domain.
+There could be hunger before the next moon!"
+        [/message]
+        [message]
+            speaker=Karron
+            message= _ "Gorlack we must--
+
+Gorlack, hatchlings do not belong at a counsel gathering."
+        [/message]
+        [message]
+            speaker=Arinexis
+            image_pos=right
+            mirror=yes
+            message= _ "We are <b>not</b> hatchlingss..."
+        [/message]
+        [message]
+            speaker=Arinexis
+            image_pos=right
+            mirror=yes
+            message= _ "Not hatchlings!"
+        [/message]
+        [message]
+            speaker=Zedrix
+            image_pos=right
+            mirror=yes
+            message= _ "Indeed, I am our pack'ss Elder. Ssuch longevity wandering starss bestow me. For over 11 yearsss have I lived."
+        [/message]
+        [message]
+            speaker=Reshan
+            image_pos=right
+            mirror=yes
+            message= _ "They are ‘Saurians’."
+        [/message]
+        [message]
+            speaker=Karron
+            message= _ "Their likeness to hatchlings is disarming. Disturbingly so. I will not have them near my fli--"
+        [/message]
+        [message]
+            speaker=Gorlack
+            image_pos=right
+            mirror=yes
+            message= _ "Enough."
+        [/message]
+        [message]
+            speaker=Gorlack
+            image_pos=right
+            mirror=yes
+            message= _ "I shall have a closer survey of the greatland focused on game abundance.
+
+Reshan, see to it at once."
+        [/message]
+        [message]
+            speaker=Reshan
+            image_pos=right
+            mirror=yes
+            message= _ "By your will."
+        [/message]
+        [move_unit]
+            id=Reshan
+            to_x,to_y=33,4
+        [/move_unit]
+        [put_to_recall_list]
+            id=Reshan
+        [/put_to_recall_list]
+        [message]
+            speaker=Karron
+            message= _ "Gorlack, arriving here, I noticed a large gathering of game to the northeast.
+A nest of haldric."
+        [/message]
+        [message]
+            speaker=Zedrix
+            image_pos=right
+            mirror=yes
+            message= _ "You ssaw Demon King?! You ssssaw <b>The Haldric</b>?"
+        [/message]
+        [message]
+            speaker=Karron
+            message= _ "Gorlack, your hatchling-things cry nonsense. You must gain control over them."
+        [/message]
+        [message]
+            speaker=Arinexis
+            image_pos=right
+            mirror=yes
+            message= _ "Not nonsensse! Creaturess you sssaw call themsselvesss “Humanss”. Lumbering, brutish, odiouss monstersss covered in moisst sskin of toad. Hirelingsss sssummoned from underssside of world by elvesss of deep, dark foresst-- Wesssmere foressst.
```

Comment:
Humans should be lower-case.

Reply:
The multiplied S's here don't sit right with me. I can't quite put my finger on why though… maybe the problem is just that a double S actually means something? In other words, maybe try to make sure every S is at least tripled?

But also, it might make more sense if not all the S's are multiplied… if you try to imagine them actually talking, I think you'd find that some S's make more sense to be extended than others. It's hard to explain why that would be though… possibly related to accenting? Not sure. Also, possibly related to voicing – a voiced S feels less extendable than an unvoiced S (though that doesn't mean it can't be extended, obviously).

On another unrelated note, I don't like how he seems unable to speak in full sentences.

Reply:
Good point for avoiding double S's, I agree.

Not sure what the rules are for which S's get extended or not in the saurian speech of other campaigns. Accenting might be the best way to go?

The saurian characters are speaking in sentence fragments in this instance because they have lost some emotional composure over the subject matter. They normally speak in full sentences, but without using "the" and "a" at all, which is maybe something I should change?

Reply:
For a comparison, see the variance in the number of esses in the TVTropes article on this speech pattern: https://tvtropes.org/pmwiki/pmwiki.php/Main/SssssnakeTalk

Reply:
So what we have so far:
* Three S's being the minimum.
* No elongation for S's that sound like Z.
* Going for greater than three S's for words being accented.

Reply:
My original request regarding 'humans' seems to be resolved. Not sure if this should be left opened for the discussion about the saurian speech pattern.

Reply:
> maybe the problem is just that a double S actually means something? In other words, maybe try to make sure every S is at least tripled?

After some testing, this turned out to be excellent advice, making the saurian accent quite a lot more readable. Applied.

> if you try to imagine them actually talking, I think you'd find that some S's make more sense to be extended than others. [...] a voiced S feels less extendable than an unvoiced S (though that doesn't mean it can't be extended, obviously).

Likewise this turned out really well, making the text read and sound quite a lot more natural. Applied.

At some point it might be worth applying these rules to the saurian speech in all mainline campaigns.